src/test/ui/rfc-2632-const-trait-impl/issue-102156.rs

```diff
@@ -0,0 +1,15 @@
+#![feature(allocator_api)]
+#![feature(const_trait_impl)]
+
+use core::convert::{From, TryFrom};
+//~^ ERROR
+//~| ERROR
```

Comment:
Maybe add the error code to make it a tad clearer, or is it mostly unnecessary for this case. Not sure on the error annotations guidelines.

Reply:
The errors are expected, but checking the error message is mostly unnecessary because we are only ensuring that it doesn't ICE.